include/cuco/detail/static_map/static_map_ref.inl

```diff
@@ -411,14 +427,17 @@ class operator_impl<
    * element that prevented the insertion) and a `bool` denoting whether the insertion took place or
    * not.
    *
+   * @tparam Value Input type which is implicitly convertible to 'value_type'
+   *
    * @param group The Cooperative Group used to perform group insert_and_find
    * @param value The element to insert
    *
    * @return a pair consisting of an iterator to the element and a bool indicating whether the
    * insertion is successful or not.
    */
+  template <typename Value>
   __device__ thrust::pair<iterator, bool> insert_and_find(
-    cooperative_groups::thread_block_tile<cg_size> const& group, value_type const& value) noexcept
+    cooperative_groups::thread_block_tile<cg_size> const& group, Value const& value) noexcept
```

Comment:
The non-CG `insert_and_find` needs to be updated as well.

Reply:
Addressed in [63e1fae](https://github.com/NVIDIA/cuCollections/pull/381/commits/63e1fae7af04eafeaa737ba175bfd87d0f51fc1f)